From c2f4df405475c6cb432a05feca5a663bc005cbf4 Mon Sep 17 00:00:00 2001
From: Pooja Prajod <a0132412@ti.com>
Date: Fri, 15 May 2015 20:20:55 +0530
Subject: [gst-ducati] gstducati: Correct h264 ducati plugin caps

Due to some typo, h264 caps was missing level=5 information.
Applying this patch enables playing of h264 streams with level=5

Signed-off-by: Pooja Prajod <a0132412@ti.com>
---
 src/gstducatih264dec.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/gstducatih264dec.c b/src/gstducatih264dec.c
index bae30b8..d196fc3 100644
--- a/src/gstducatih264dec.c
+++ b/src/gstducatih264dec.c
@@ -96,7 +96,7 @@ static GstStaticPadTemplate sink_factory = GST_STATIC_PAD_TEMPLATE ("sink",
         "framerate = (fraction)[ 0, max ],"
         "profile = (string) {high, high-10-intra, high-10, high-4:2:2-intra, "
         "high-4:2:2, high-4:4:4-intra, high-4:4:4, cavlc-4:4:4-intra}, "
-        "level = (string) {1, 1b, 1.1, 1.2, 1.3, 2, 2.1, 2.2, 3, 3.1, 3.2, 4, 4.1, 4.2, 5.1};")
+        "level = (string) {1, 1b, 1.1, 1.2, 1.3, 2, 2.1, 2.2, 3, 3.1, 3.2, 4, 4.1, 4.2, 5, 5.1};")
     );
 
 static const struct
@@ -447,8 +447,7 @@ gst_ducati_h264dec_class_init (GstDucatiH264DecClass * klass)
 }
 
 static void
-gst_ducati_h264dec_init (GstDucatiH264Dec * self,
-    gpointer gclass)
+gst_ducati_h264dec_init (GstDucatiH264Dec * self, gpointer gclass)
 {
 #ifndef GST_DISABLE_GST_DEBUG
   GstDucatiVidDec *dec = GST_DUCATIVIDDEC (self);
-- 
1.7.9.5

